dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -617,10 +619,15 @@ isNormalizedIsConsistentWithNormalize expression =
         Nothing -> Test.QuickCheck.discard
         Just v -> v
   where
+      denotedExpression :: Expr Void Import
+      denotedExpression = Dhall.Core.denote expression
+
       maybeProp = do
           nf <- Control.Spoon.spoon (Dhall.Core.normalize expression)
           isNormalized <- Control.Spoon.spoon (Dhall.Core.isNormalized expression)
-          return $ isNormalized === (nf == expression)
+          -- Dhall.Core.isNormalized ignores 'Note's and other annotations.
+          -- So we do the same when checking the result of 'normalize'.
+          return $ isNormalized === (nf == denotedExpression)
```

Comment:
I went with this simple test fix in order to keep the focus on the pretty-printer changes.